### ISSUE TITLE
HTML section listed 2nd in Map

### ISSUE THREAD

**Issue by kprenesti** [freeCodeCamp/freeCodeCamp#12880]
Challenge [say-hello-to-html-elements](http://beta.freecodecamp.com/en/challenges/basic-html-and-html5/say-hello-to-html-elements) has an issue.
User Agent is: <code>Mozilla/5.0 (Macintosh; Intel Mac OS X 10_11_6) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.95 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

HTML section listed 2nd in map as opposed to first.  You must have HTML knowledge before CSS will make a lick of sense.

```html


<h1>Hello</h1>

```

**Comment by ajain17** [comment]
I think it's a good idea to enlist HTML before CSS in the map. Should I change the ordering of the courses? I fixed it locally can create a PR if needed.

**Comment by raisedadead** [comment]
@ajain17 thanks please check for open PR's, I believe there is one already.

**Comment by ajain17** [comment]
@raisedadead  I dont see one for this issue.

**Comment by Manish-Giri** [comment]
I guess this change is done in https://github.com/freeCodeCamp/freeCodeCamp/pull/12663 already?

**Comment by raisedadead** [comment]
@Manish-Giri thanks. @ajain17 thanks as well, let me check on staging, and will review your pull request as well.

**Comment by HKuz** [comment]
For what it's worth, the original issue for the ordering was #12646 - that was closed automatically when PR #12663 was QA'd and merged. This should be fixed on `staging` already (I see it ordered with HTML first, then CSS when I run it locally) - but I worked on the PR, so if someone else can confirm, much appreciated!

I'm not sure when the next push of `staging` is to beta, so this shows up for everyone helping with the QA process.
![screen shot 2017-01-26 at 11 05 03 am](https://cloud.githubusercontent.com/assets/9591826/22339313/85857bd2-e3b7-11e6-8b78-2b48ffd5443a.png)
